railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -54,7 +54,7 @@ def self.plugin_rails_options(opts, options)
 
     options[:color] = true
     options[:output_inline] = true
-    options[:patterns] = opts.order!
+    options[:patterns] = opts.order! unless defined?(@rake_patterns)
```

Comment:
Would rather have the ivar override here, when it's defined, and remove the check down below.